src/icq/poller.rs

```diff
@@ -99,12 +99,30 @@ pub async fn process_events(
     }
 }
 
-pub async fn process_event_buddy_list(event_data: &events::BuddyListData) {
+pub async fn process_event_buddy_list(
+    mut tx: FdSender<SystemMessage>,
+    account_info: &AccountInfo,
+    event_data: &events::BuddyListData,
+) {
     for group in &event_data.groups {
         for buddy in &group.buddies {
             match &buddy.user_type {
                 events::UserType::Chat => {
-                    // Do we already have a chat?
+                    let chat_info = ChatInfo {
+                        sn: buddy.aim_id.clone(),
+                        stamp: None,
+                        title: match &buddy.friendly {
+                            Some(friendly) => friendly.clone(),
+                            None => buddy.aim_id.clone(),
```

Comment:
Could `display_id` be used here?

Reply:
AFAIK it never happens that we use aim_id. Friendly is always present in this case.

Also in the data that I have captured display_is is always the same as aim_id.